projects/plugins/jetpack/extensions/blocks/donations/first-time-modal.js

```diff
@@ -0,0 +1,62 @@
+import { Modal, Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+const FirstTimeModal = ( { onClose } ) => {
+	return (
+		<Modal
+			className="jetpack-donations-first-time-modal"
+			onRequestClose={ onClose }
+			title={ __( 'Accept Donations with Stripe', 'jetpack' ) }
+		>
+			<div className="jetpack-donations-first-time-modal__content">
+				<p>
+					{ __(
+						"To accept donations on your site, you'll need to connect your Stripe account. Here's what you need to do:",
+						'jetpack'
+					) }{ ' ' }
+					<a
+						href="https://wordpress.com/support/wordpress-editor/blocks/donations/#about-donations"
```

Comment:
Would be better if you used https://jetpack.com/support/jetpack-blocks/donations-block/ for Jetpack sites and above link for WP.com sites. Even better to rely on `getRedirectUrl` from `@automattic/jetpack-components` as is typical at Jetpack. :-)

Reply:
Thanks @simison I'll add these fixes next week (after a support rotation)!

Reply:
Nothing urgent, thanks! :-)